src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.model;
+
+public abstract class Figure {
+    protected String name;
+    protected double area;
```

Comment:
just 1 more thing: you are passing the already calculated area into the constructor

but this can be done in code for each figure

all figures have area but can we make Figure class implement interface AreaCalculator with method calculateArea() instead of having the field? each Figure implementation can have its own method version, and Figure class itself is abstract, so it doesn't have to override this method

Reply:
fixed, added implementations getArea